addons/im_livechat/static/src/legacy/models/website_livechat.js

```diff
@@ -80,6 +82,24 @@ var WebsiteLivechat = AbstractThread.extend(ThreadTypingMixin, {
     getUUID: function () {
         return this._uuid;
     },
+    /**
+     * @returns {boolean}
+     */
+    getHasOperator() {
+        return Boolean(this._operatorPID);
+    },
+    /**
+     * @returns {number}
+     */
+    getChatbotScriptId() {
+        return this._chatbotScriptId;
+    },
+    /**
+     * @returns {boolean}
+     */
+    getIsTemporary: function() {
+        return this._isTemporary;
+    },
```

Comment:
These getters are useless.
Fix would have less diff by just looking at the attributes.
We don't really care of `_` prefix.